src/Installation/guides/package.md

```diff
@@ -0,0 +1,30 @@
+# Installation from your package manager
+
+### Arch Linux or derivatives (AUR)
+
+To install Vinegar, run the following commands (where "version" is the current version of Vinegar):
+
+```sh
+git clone https://aur.archlinux.org/vinegar.git
+cd vinegar
+makepkg -si
+pacman -U vinegar-version-x86_64.pkg.tar.zst
```

Comment:
`makepkg -i` installs the package, the pacman command here is redundant.

Reply:
additionally, you can mention an aur helper being used instead.

Reply:
> `makepkg -i` installs the package, the pacman command here is redundant.

Oh okay I thought it only built it